src/wallet/wallet2.h

```diff
@@ -635,6 +648,9 @@ namespace tools
         FIELD(dests)
         FIELD(construction_data)
         FIELD(multisig_sigs)
+        if (version < 1)
+          return true;
+        FIELD(multisig_tx_key_entropy)
```

Comment:
This will not be initialized after loading a v0 record. Might be OK in practice, or setting to crypto::null_hash in the early out would be needed.

Reply:
Hmm ok, setting to `crypto::null_hash` then asserting it is not null in the multisig code would be good. By early out do you mean like this?
```
if (version < 1)
{
  multisig_tx_key_entropy = crypto::null_hash;
  return true;
}
FIELD(multisig_tx_key_entropy)
```

Reply:
Yes (might or might not be better to do the assignment if loading only, I don't really mind either way)